server/lib/models/rescue.go

```diff
@@ -18,7 +18,7 @@ func InitRescue() {
 	create_table(Rescue{}, "rescues", "Id").AddIndex("account", "INDEX", []string{"Account"})
 }
 
-func NewRescue(dbif Dbif, account string) (*Rescue, error) {
+func NewRescue(dbif Dbif, account proto.UUID) (*Rescue, error) {
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [golint] exported function NewRescue should have comment or be unexported